bootstrap-joining-demo/src/main/scala/akka/cluster/bootstrap/DemoApp.scala

```diff
@@ -31,7 +29,7 @@ object DemoApp extends App {
 
   log.info(s"Started [$system], cluster.selfAddress = ${cluster.selfAddress}")
 
-  ClusterHttpManagement(cluster).start()
+  AkkaManagement(system).start()
   ClusterBootstrap(system).start()
```

Comment:
the first line actually knows about the routes from bootstrap, and the 2nd line starts the probing and DNSing around here

Reply:
This is done so we only have one "management http server"